hive/pom.xml

```diff
@@ -133,5 +133,10 @@
             <version>${junit.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>30.1.1-jre</version>
```

Comment:
Let's issue a separate PR in case we need to revert. 
Also let's get the version from common here: https://github.com/confluentinc/common/blob/5.5.x/pom.xml#L63